torch/nn/quantized/functional.py

```diff
@@ -115,12 +115,12 @@ def conv2d(input, weight, bias,
     padding = _pair(padding)
     dilation = _pair(dilation)
 
-    prepacked_weight = ops.quantized.fbgemm_conv_prepack(
+    prepacked_weight = _ops.quantized.fbgemm_conv_prepack(
         weight.permute([0, 2, 3, 1]), stride, padding, dilation, groups)
-    return ops.quantized.fbgemm_conv2d(input.permute([0, 2, 3, 1]),
-                                       prepacked_weight, bias,
-                                       stride, padding, dilation,
-                                       groups, scale, zero_point).permute([0, 3, 1, 2])
+    return _ops.quantized.fbgemm_conv2d(input.permute([0, 2, 3, 1]),
```

Comment:
maybe remove `as _ops` in the import?

Reply:
The reason it was there was so that the user wouldn't have `ops` in their top level after `from torch.quantized.functional import *`

Reply:
maybe use torch.ops directly then?

Reply:
Does it make a difference? @zafartahirov  do you know?